core/app/models/spree/order.rb

```diff
@@ -787,7 +787,7 @@ def after_cancel
     end
 
     def send_cancel_email
-      OrderMailer.cancel_email(self.id).deliver_later
+      OrderMailer.cancel_email(self).deliver_later
```

Comment:
@jordan-brough I'm thinking that you might want to monkey patch it. This way Solidus remains closer to the sugar that Rails provides and helps you find reason to remove the ResqueMailer.